Detector/DetCommon/src/DetUtils.cpp

```diff
@@ -145,6 +145,12 @@ std::array<uint, 2> numberOfCells(uint64_t aVolumeId, const DD4hep::DDSegmentati
   uint phiCellNumber = aSeg.phiBins();
   // get segmentation cell width in eta
   double etaCellSize = aSeg.gridSizeEta();
+  // eta segmentation calculate maximum eta from the inner radius, offset is taken into account
+  /*
```

Comment:
Does that mean we no longer take offset into account?
Also, commented code should be removed.

Reply:
I decided to remove my changes, as it will not calculate the number of cells in the endcaps or forward because of their special shape. I'd propose to leave it as it was and think about it once more in the next PR.